torch/_inductor/kernel/mm.py

```diff
@@ -153,8 +136,7 @@ def parse_version(version_string: str) -> Optional[tuple[int, ...]]:
     # inductor generates a suffix
     {{store_output(("idx_m", "idx_n"), "acc", "mask")}}
 """
-        if (torch.version.hip is None)
-        or (has_triton and triton_major >= 3 and triton_minor >= 3)
+        if (torch.version.hip is None) or (has_triton and triton_version >= "3.3.0")
```

Comment:
Since you always initialize trition version no need to check if has_triton is true
```suggestion
        if (torch.version.hip is None) or triton_version >= "3.3.0"
```